barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_verifier.cpp

```diff
@@ -35,123 +36,106 @@ void ProtogalaxyVerifier_<DeciderVerificationKeys>::run_oink_verifier_on_each_in
     }
 }
 
+template <typename FF, size_t NUM>
```

Comment:
This was just computed inline; I make it a function.

Reply:
I think, this computation could re-use some methods from polynomial_arithmetic/univariate/polynomial classes. Not necessary in terms of efficiency, but the code would look cleaner. Could do that in my refactoring work.